src/mocks/data.ts

```diff
@@ -4808,6 +4808,34 @@ export const mockNewsPods = {
     totalResults: 5
 };
 
+const promoPodTag = "promo";
```

Comment:
Unless I'm misunderstanding this, this constant does not add any value: you can simply set the string directly in the only place that is being used. It's clear that this is a promo tag even without reading the variable name.